build-contracts/node-rdkafka-test/test-install.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+npm link node-rdkafka
```

Comment:
We could also remove this line and do the symlink manually from inside the yolean/node-rdkafka image directly. But then all downstream images would need to use the very same workdir I guess

Reply:
I read about by-design npm link, but I thought I got rid of that with `NODE_PATH` when [require](https://github.com/Yolean/docker-base/pull/3/files#diff-44fb691061cd5bcaff1722e66cab9422R35) worked in the first test. Is it that `npm install` will try to install the lib unless it is actually present in node_modules?

Reply:
Yeah, without this, if one dependency includes node-rdkafka (or snappy), it'll get installed and rebuilt anyways.

Reply:
Ok, sounds good to have `npm link` in `Dockerfile` as a way to declare the assumption anyway.